e2e/time-release/timeRelease.test.ts

```diff
@@ -50,4 +50,92 @@ describe('TimeRelease', function () {
       assert.notEqual(target, undefined, 'should have returned ReleaseScheduleAdded event');
     });
   });
+
+  describe('create schedule and cancel transfer', function () {
+    it('create a schedule transfer', async function () {
```

Comment:
nit: copypasta -  has the same name as the test below
```suggestion
    it('creates and cancels named scheduled transfers', async function () {
```